authN-authZ/auth-apisix/README.md

```diff
@@ -2,13 +2,15 @@
 
 Follow the steps to enable authentication and authorization of OPEA services using APISIX api gateway and Identity provider like keycloak
 
-## Prerequisites
+## Start ChatQnA service
 
-1. Make sure ChatQnA service is running and accessible locally
+Please refer to [GenAIExamples ChatQnA](https://github.com/opea-project/GenAIExamples/tree/main/ChatQnA/kubernetes/intel) to start `chatqna` megaservice.
 
-2. Run keycloak, setup a realm with OIDC based authentication and add users with passwords for each user.
+## Start Keycloak and configuration
 
-Steps to start keycloak from official keycloak helm chart
+In this step, we run keycloak, setup a realm with OIDC based authentication and add a user with password.
```

Comment:
Since you are changing the doc here, can you help to add some description on the testing scenario?

Reply:
What does "testing scenario" refer to? As I have added the detail steps below.

Reply:
> What does "testing scenario" refer to? As I have added the detail steps below.

Just to declare the test cases, e.g. if you would like to test that only authenticated user could access the service. You defined two cases: Mary who is an authenticated user who is part of 'user' group can access with password, Bob who is an anonymous user cannot access the service.

Reply:
Thanks for clarification, I have added the description